main.go

```diff
@@ -10,8 +10,12 @@ func main() {
 	mx := http.NewServeMux()
 
 	logFile, _ := os.Create("log.log")
-	core := Core{lg: slog.New(slog.NewJSONHandler(logFile, nil))}
-	api := API{core: &core}
-
+	core := Core.Core{Lg: log.New(logFile, "", log.LstdFlags)}
+	api := Api.API{Core: &core}
+	api.Core.Sessions = make(map[string]string)
+	api.Core.Users = make(map[string]User.User)
```

Comment:
Давай чуть структурируем мешанину

```suggestion
	core := Core.Core{
	    Lg: log.New(logFile, "", log.LstdFlags),
	    Sessions: make(map[string]string),
	    Users: make(map[string]User.User)
	}

	api := Api.API{Core: &core}
```